utils/firestore.js

```diff
@@ -1369,16 +1463,21 @@ const getLocations = async (institute) => {
 
 const searchBoxes = async (institute, flag) => {
     let snapshot = ``
-    if ((institute === nciCode || institute == nciConceptId) && flag === `bptl`) {
-        snapshot = await db.collection('boxes').get()
-    } 
-    else { 
+    if (institute === nciCode || institute == nciConceptId) {
+        if (flag === `bptl`) {
+            snapshot = await db.collection('boxes').get();
```

Comment:
NIT: We cound define `const boxesCollection = db.collection('boxes')`
Then 
`snapshot = await boxesCollection.get()`
`snapshot = await .where(fieldMapping.submitShipmentFlag.toString(), "==", fieldMapping.yes)
                            .where(fieldMapping.siteShipmentReceived.toString(), "==", fieldMapping.no).get();`

Reply:
Thanks for the feedback. I agree with this change since I am repeating `db.collection('boxes')` twice.